<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSMSClass</class>
 <widget class="QMainWindow" name="GSMSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GSMS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1061</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="managePage">
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>421</width>
        <height>561</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="managePage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>417</width>
         <height>505</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Selling</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Brand：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Type：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Price：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Discount：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Number：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Sum Price：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sellOkBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>350</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Deal!</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sellCancelBtn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>350</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Not now</string>
        </property>
       </widget>
       <widget class="QComboBox" name="sellBrandComboBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QComboBox" name="sellGnameComboBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellPriceLineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellDisLineEdit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sellNumSpinBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>220</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellSumLineEdit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>270</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="sellLastNumLabel">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>417</width>
         <height>505</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Improt</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Brand:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="importBrandComboBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="importGnameComboBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Number:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="importNumSpinBox">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>70</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="importOkBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
       <widget class="QPushButton" name="importCancelBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>150</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Mistake</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="TestBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stock</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>551</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sellListBtn">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>480</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Sell_List</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="statisticsPage">
     <widget class="QPushButton" name="sortBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SORT</string>
      </property>
     </widget>
     <widget class="QComboBox" name="staBrandCombobox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
     </widget>
     <widget class="QTableView" name="statableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>811</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Brand</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="staGnameCombobox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="changeBtn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>220</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CHANGE (PRICE)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delBtn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>80</y>
        <width>121</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>DELETE (SID)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="resultLineEdit">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>160</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>160</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExtra_2">
    <property name="title">
     <string>Extra</string>
    </property>
    <addaction name="actionManage"/>
    <addaction name="actionStatistics"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuExtra_2"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionManage"/>
   <addaction name="actionStatistics"/>
   <addaction name="actionQuit"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Resources/help_normal.png</normaloff>:/new/icon/Resources/help_normal.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Resources/logout_normal.png</normaloff>:/new/icon/Resources/logout_normal.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Resources/modify_normal.png</normaloff>:/new/icon/Resources/modify_normal.png</iconset>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionManage">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/Resources/skills_normal.png</normaloff>:/new/icon/Resources/skills_normal.png</iconset>
   </property>
   <property name="text">
    <string>Manage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
